src/model/Sound.ts

```diff
@@ -9,6 +9,7 @@ export type Sound = {
   extension: string;
   fileHash: string;
   fileName: string;
+  tags?: string[];
```

Comment:
no reason for tags to be able to be undefined, they can be an empty array instead

```suggestion
  tags: string[];
```